index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>YesufFenta Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;500&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet" />
+    </head>
+<body>
+    <img
+    src="assets/images/Light.svg" alt="status-bar" class="status-bar"/>
+    <header class="header">
```

Comment:
The top bar seen on the figma was only there to present the design on a mobile phone and it is not necessary for you to design it, so consider removing it to ensure that your UI is as expected. I mean the `img` before the header

Reply:
Great Thanks

Reply:
You are welcomed

Reply:
Awesome, thanks for your feedback that helps me to improve and know technical skills.